tests/testthat/test-sprintf_linter.R

```diff
@@ -91,3 +91,42 @@ test_that("edge cases are detected correctly", {
     linter
   )
 })
+
+local({
+  linter <- sprintf_linter()
+  unused_fmt_msg <- "too few arguments"
+  unused_arg_msg <- "one argument not used by format"
+  pipes <- pipes(exclude = "%$%")
+  patrick::with_parameters_test_that(
+    "piping into sprintf works",
+    {
+      expect_lint(paste("x", pipe, "sprintf(fmt = '%s')"), NULL, linter)
+      # no fmt= specified -> this is just 'sprintf("%s", "%s%s")', which won't lint
+      expect_lint(paste('"%s"', pipe, 'sprintf("%s%s")'), NULL, linter)
+      expect_lint(paste("x", pipe, "sprintf(fmt = '%s%s')"), unused_fmt_msg, linter)
+
+      # Cannot evaluate statically --> skip
+      expect_lint(paste("x", pipe, 'sprintf("a")'), NULL, linter)
+      # Nested pipes
+      pipe_width <- nchar(pipe)
```

Comment:
unused?

Reply:
Ah right, vestigial from defining `column_number` with that. I switched to the `nchar()` approach for readability, hopefully it's less arbitrary.